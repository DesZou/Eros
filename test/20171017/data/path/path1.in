5 8
758589575 65050098 778461380 196924662 497819799 
3 5 25912683
3 2 778176342
1 2 909547144
1 5 67940701
1 3 107927626
2 5 125207004
2 4 183459168
4 3 388695028
