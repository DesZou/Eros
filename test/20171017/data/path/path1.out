0 94980727951383300 52889211858627380 139170212282973600 51539107496792075 
94980727951383300 0 97468817119505520 36127634649201216 62330525564672196 
52889211858627380 97468817119505520 0 142815877094931840 20172022967682540 
139170212282973600 36127634649201216 142815877094931840 0 91329606138467232 
51539107496792075 62330525564672196 20172022967682540 91329606138467232 0 
